README.md

```diff
@@ -1,2 +1,218 @@
+        "url": "https://github.com/paritytech/stale-issues-finder/issues/3",
+        "title": "Help me with reading",
+        "daysStale": "18"
+    }
+]
+```
+
+### Using a GitHub app instead of a PAT
+In some cases, specially in big organizations, it is more organized to use a GitHub app to authenticate, as it allows us to give it permissions per repository and we can fine-grain them even better. If you wish to do that, you need to create a GitHub app with the following permissions:
+- Repository permissions:
+	- Issues
+		- [x] Read
+
+Because this project is intended to be used with a token we need to do an extra step to generate one from the GitHub app:
+- After you create the app, copy the *App ID* and the *private key* and set them as secrets.
+- Then you need to modify the workflow file to have an extra step:
+```yml
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+      - name: Fetch issues from here
+        id: stale
+        uses: paritytech/stale-issues-finder@main
+        with:
+          days-stale: 10
+          # The previous step generates a token which is used as the input for this action
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+```
+
+Be aware that this is needed only to read issues from **external private repositories**. 
+If the issue is in the same repository, or the target repository is public, the default `${{ github.token }}` has enough access to read the issues.
+
+## Example workflow
+
+Let's make an example. We want to have a workflow that runs every Monday at 9 in the morning and it informs through a slack message in a channel. We can also trigger it manually if we want to.
+
+This issue needs to run on 3 different repositories:
+- The current repository
+- `example/abc` repository
+- `example/xyz` repository
+
+```yml
+name: Find stale issues
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 9 * * 1'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch issues from here
+        id: local
+        uses: paritytech/stale-issues-finder@main
+        with:
+          GITHUB_TOKEN: ${{ github.token }}
+      - name: Fetch abc issues
+        id: abc
+        uses: paritytech/stale-issues-finder@main
+        with:
+          GITHUB_TOKEN: ${{ github.token }}
+          owner: example
+          repo: abc
+      - name: Fetch xyz issues
+        id: polkadot
+        uses: paritytech/stale-issues-finder@main
+        with:
+          GITHUB_TOKEN: ${{ github.token }}
+          owner: example
+          repo: xyz
+      - name: Post to a Slack channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: 'CHANNEL_ID,ANOTHER_CHANNEL_ID'
+          slack-message: "Stale issues this week: \n$LOCAL_ISSUES \n$ABC_ISSUES \n$XYZ_ISSUES"
```

Comment:
could we make an example with all our repos in https://github.com/paritytech/opstooling/tree/master/.github/workflows ?

Reply:
Yes, as soon as I have a Matrix bot I'll have it running!

I can make one with no Matrix bot but that will still work in the intended way.